openshift/Makefile

```
@@ -12,6 +12,18 @@ ifndef PROVIDER_VERSION
 	$(error PROVIDER_VERSION is undefined)
 endif
 
+verify: verify-ocp-manifests
```

checkmake (0.2.2)

[warning] Target "verify" should be declared PHONY.

(phonydeclared)